pkg/cmd/roachtest/README.md

```diff
@@ -89,7 +89,7 @@ Especially for OSX users, building a linux CockroachDB binary is an extremely ti
 # Make a single-node local cluster.
 roachprod create -n 1 local
 # Tell roachprod to place a CockroachDB 21.1.5 Linux binary onto node 1.
```

Comment:
We're still referencing 21.1.5 here.

Also, to make this more "future-proof", we can mention that you can pick a recent release from https://www.cockroachlabs.com/docs/releases/.

Reply:
Great idea, I'll add those